api/envoy/api/v2/core/protocol.proto

```diff
@@ -32,6 +32,12 @@ message UpstreamHttpProtocolOptions {
   // :ref:`router filter <config_http_filters_router>`.
   // This field is intended to set with `auto_sni` field.
   bool auto_san_validation = 2;
+
```

Comment:
Why should we need v2 API? cc @htuch

Reply:
I wasn't sure whether we are supposed to make changes to v2 or not. I can remove it.